assets/src/components/Image/ImageBlockEdit.js

```diff
@@ -12,17 +12,21 @@ export const ImageBlockEdit = (BlockEdit) => {
     }
 
     const { attributes, clientId } = props;
-    const { id, caption } = attributes;
+    const { id, caption, className } = attributes;
 
     // Get image data
     const image = useSelect(select => id ? select('core').getMedia(id) : null);
     const credits = image?.meta?._credit_text;
     // Compile data for insertion
-    const image_credits = credits && credits.length > 0 && ! caption.includes(credits)
+    const image_credits = credits && credits.length > 0 && caption && ! caption.includes(credits)
```

Comment:
Just thinking..

What do you think to change to this:
```
const credits = image?.meta?._credit_text ?? [];
```
Then
```
const image_credits = credits.length && ! caption.includes(credits);
```
And finally (from [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/864/files#diff-fa236ef56a43d882aec0fcdf53bd1cfcb08666dfc9f4f4bc5eb08649b7a04a63R26))
```
if (!caption && credits.length && (!className || !className.includes('no-caption'))) {
```
I think it will make the code more cleaner.